server/configuration.go

```diff
@@ -156,7 +156,7 @@ func (p *Plugin) diffConfiguration(newConfiguration *configuration) {
 		}
 
 		if _, err := p.API.CreatePost(&model.Post{
-			UserId:    newConfiguration.demoUserId,
```

Comment:
Is there an opportunity to remove some of the demo user configuration?

Reply:
Or we could keep the demo user configuration but patch the bot, when it changes.

Reply:
It was the only example of using a user there so I left it. Maybe a separate ticket to show a slightly different use case?